packages/fxa-payments-server/src/lib/l10n.test.js

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint jest/valid-expect: 0 */
+
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import waitUntil from 'async-wait-until';
+import { Localized } from 'fluent-react';
+import sinon from 'sinon';
+
+import fetchMock from 'fetch-mock';
+import AppLocalizationProvider from './l10n';
+
+
+describe('<AppLocalizationProvider/>', () => {
+  const locales = ['en-US', 'es-ES'];
+  const bundles = ['greetings', 'farewells'];
+  function waitUntilTranslated() {
+    return waitUntil(() => {
+      return AppLocalizationProvider.prototype.render.callCount === 2;
```

Comment:
Super minor, but can we get rid of `{ return` and simply `=> AppLocalizationProvider...`?

Reply:
fixed